runtime/compiler/ilgen/J9ByteCodeIlGenerator.hpp

```diff
@@ -238,6 +238,7 @@ class TR_J9ByteCodeIlGenerator : public TR_IlGenerator, public TR_J9ByteCodeIter
    void         handleSideEffect(TR::Node *);
    bool         valueMayBeModified(TR::Node *, TR::Node *);
    TR::Node *    genCompressedRefs(TR::Node *, bool genTT = true, int32_t isLoad = 1);
+   void         unresolvedValueTypeOpAbort(const char* bytecodeName, const char* refType);
```

Comment:
I'm not a fan of this name but I couldn't think of anything else off the top of my head. I'm open to suggestions if anyone has a better name.

Reply:
This isn't much better, but how about making it imperative?  Something like `abortOnUnresolvedValueType` or `abortForUnresolvedValueTypeOp`

Reply:
Moving abort to the start of the method name would aid readability IMO - I'm not sure if we want to make it more obvious with an ABORTForUnresolvedValueTypeOp or something like that.

Reply:
Alright, I'll change it to `abortForUnresolvedValueTypeOp`. I don't think there's a particular need to make it stand out because there's really nothing special about it.